### DISCUSSION TITLE
Gobel Power 280Ah 15kWh Batterie - Hat jemand Erfahrung?

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#558] [Q&A]
2 davon = 30kWh! (verlockend)

[Gobel Power GP-SR1-PC200 Standard 51.2V 280Ah 15kWh LiFePO4 Server Rack Battery](https://www.gobelpower.com/choose-your-battery-gobel-power-gpsr1pc200-standard-512v-280ah-15kwh-lifepo4-server-rack-battery_p120.html#3)

Ich lese, dass die Gobel Batterien RJ45 & RS485 Kompatibel sind mit Pylontech: 
CAN Port: 1.Goodwe 2.Growatt 3.Luxpower 4.Pace 5.PV 6.**Pylontech** 7.SE 8.SMA 9.Sofar 10.SRD 11.Studer 12.Victron 13. Deye 

RS485 Port: 1.Deye 2.Growatt 3.Lithium 4.Luxpower 5.MSL 6.Pace 7.Phocos 8.**Pylontech**  9.SE 10.Solar 11.Voltron

Hat jemand Erfahrung ob sie mit der OpenDTUonBattery funktionieren?

**Comment by Leviathan09** [comment]
Würde mich auch interessieren. Bzw ich würde die Frage etwas erweitern.
Wie sieht es generell mit der Kompatbilität von BMSen aus die das Pylontech Protokoll unterstützen? z.B. Pace BMS

**Reply by alexseuf** [reply]
Mit pace BMS über CAN hat es bei mir testweise auf Anhieb funktioniert

**Reply by Manos1966** [reply]
@alexseuf du meist, du hast mit der OpenDTUonBattery ueber CAN zugegriffen?

**Comment by Manos1966** [comment]
Ich habe eine GOBEL GP-SR1-RV150 Batterie Bj.2022 getestet.
Sogar in DEYE Konfiguration, der CANbus hat funktioniert.
![GOBEL CANBus B - Copy](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/dd00c690-20be-43a1-9523-d395bcb60c33)
